apps/stats-mainnet/src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="assets/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Health check for Vega's mainnet" />
+    <link rel="apple-touch-icon" href="assets/apple-touch-icon.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="assets/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Vega Mainnet Stats</title>
+    <script>
+      // Set dark mode preference indicator here to avoid FOUC
+      if (
+        localStorage.theme === 'dark' ||
+        (!('theme' in localStorage) &&
+          window.matchMedia('(prefers-color-scheme: dark)').matches)
+      ) {
+        document.documentElement.classList.add('dark-mode-preferred');
+      } else {
+        document.documentElement.classList.remove('dark-mode-preferred');
+      }
```

Comment:
chore (blocking): A hook now exists for this

Reply:
Does the hook avoid a FOUC? I did this intentionally at this early stage

Reply:
It does not avoid a FOUC, but it should be up to the hook to fix it. 

Happy to ticket this as a follow up in that case if fixing the hook would be a lot of work and we don't want a FOUC right now. I think @campbellssource noticed this when we were looking at the Trading UI on Friday?

Reply:
Sounds good, I'd like to get this on block explorer asap